src/components/on-this-page/examples/basic.js

```diff
@@ -7,7 +7,7 @@ import OnThisPage from '../on-this-page';
 export default class Basic extends React.PureComponent {
   render() {
     return (
-      <div className="limiter">
+      <div className="wmax1200 w-11/12-mm w-5/6-ml mx-auto px24 px0-mm">
```

Comment:
Following the recommendation in the migration guide -> https://github.com/mapbox/assembly/blob/publisher-staging/MIGRATION_GUIDES.md#limiter-removal